src/playwright/fixtures/mount.ts

```diff
@@ -1,52 +1,71 @@
 import { promises as fs } from 'node:fs';
 import { test } from '@playwright/test';
 import { type TemplateResult } from 'lit';
-import { render } from '@lit-labs/ssr';
-import { collectResult } from '@lit-labs/ssr/lib/render-result.js';
+import serialize from 'serialize-javascript';
 import customElements from '../../../custom-elements.json' with { type: 'json' };
 import { v8CoverageAttachmentName } from './../constants.js';
 
 export default test.extend<{
-  mount: (template: TemplateResult) => Promise<void>;
+  mount: <Type>(
+    template: (context: Type) => TemplateResult | Promise<TemplateResult>,
+    context?: Type extends void ? void : Type,
+  ) => Promise<void>;
 }>({
   async mount({ page }, use, testInfo) {
-    await use(async function mount(template: TemplateResult): Promise<void> {
+    await use(async function mount<Type>(
+      template: (context: Type) => TemplateResult | Promise<TemplateResult>,
+      context?: Type extends void ? void : Type,
+    ): Promise<void> {
       return test.step('mount()', async () => {
-        for (const string of template.strings) {
-          const hasEventListener = /@\w+=$/.test(string);
-
-          if (hasEventListener) {
-            throw new Error(
```

Comment:
No need to throw. Inline event listeners now work, though `addEventListener()` and `toDispatchEvents()` are much easier to use. So we'll likely never one.